README.md

```diff
@@ -124,18 +124,15 @@ This CLI component of this package is intended to be used with NPM scripts. It f
 ## Contributing
 
 ### Releases
-This information is for project maintainers:
-
-Requirements
 
-- Have access to [`npmjs`](https://www.npmjs.com/) Shopify organization
-- Yarn must be installed
+This information is for project maintainers:
 
 Instructions
 
-- Bump the version in [`package.json`](https://github.com/Shopify/node-themekit/blob/08d421f67adf1e5c61a6bd600e40c3e0f3924216/package.json#L3)
-- Create a release on GitHub as well as tagging the release
-- Publish the package on [`npmjs`](https://www.npmjs.com/)
+- Use [`npm version`](https://docs.npmjs.com/cli/version) to update the version.
```

Comment:
nit: If we're committing and tagging the release separately, it'd be good to note that the [`--no-git-tag-version`](https://docs.npmjs.com/cli/version#git-tag-version) flag should be passed in here.

Reply:
I *think* Shipit handles this since it does for Ruby gems, but I haven't confirmed. Once I do, I'll come back and update this.